src/experimental/safe_html_transform/safe_html.py

```diff
@@ -0,0 +1,2630 @@
+import logging
```

Comment:
This comment was meant for the safe_html.py file. Not specifically to
the logging import. This pull request is about the control panel only.
No transform code should be included here (for now).

Am 18.06.2015 um 09:56 schrieb prakhar:
> In src/experimental/safe_html_transform/safe_html.py
> <https://github.com/collective/experimental.safe_html_transform/pull/22#discussion_r32706288>:
> 
>> @@ -0,0 +1,2630 @@
>> +import logging
> 
> @tisto <https://github.com/tisto> So we have to remove all the condition
> from safe_html that include the logging functionalities. As of now we
> are assuming that our safe_html is registered, right ?
> 
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/collective/experimental.safe_html_transform/pull/22/files#r32706288>.
>